controllers/events.js

```diff
@@ -22,21 +23,17 @@ const EventsController = {
       next(err);
     }
   },
-  batchShow: async (req, res, next) => {
-    try {
-      EventsRepository.batchFind().pipe(res);
-    } catch (err) {
-      next(err);
-    }
-  },
   create: async (req, res, next) => {
     try {
       const { body: newItem } = req;
-      const isEventExist = await EventsRepository.findOne(newItem.id);
-      if (isEventExist) {
-        return res.status(409).send('Event already exists!')
-      }
       const createdItem = await EventsRepository.create(newItem);
+      if (newItem.participantsIds) {
+        const eventsUsersData = newItem.participantsIds
+          .map((participantId) => ({ userId: participantId, eventId: createdItem.id }));
+        // don't need to wait until finish, can proceed next
+        console.log('eventsUsersData', eventsUsersData)
+        EventsUsersRepository.bulkCreate(eventsUsersData)
```

Comment:
I'd recommend to wait here. This could fail (network issues, for example). Also, I'd recommend to use transaction for all this calls. Else we can have some data inconsistency due to random errors.